WalletWasabi.Fluent/Models/Wallets/CoinListModel.cs

```diff
@@ -16,6 +16,8 @@ public abstract partial class CoinListModel : IDisposable
 {
 	private readonly CompositeDisposable _disposables = new();
 
+	private object Lock { get; } = new();
```

Comment:
Lock is not used here.

Reply:
Thanks!

Reply:
Here is a slightly different fix, functionally almost the same as this. We addressed the following issues:

- New coins created by the new transaction were not in the list when pocked are refreshed. First, the coins need to be updated and then after the pockets. 
- Full list of coins passed down instead of the selection. It does not directly cause any issues, but better to pass down only the selection to the algorithms. 

https://github.com/GingerPrivacy/GingerWallet/pull/70